### ISSUE TITLE
Fan on/off temperatures not configurable

### ISSUE THREAD

**Issue by pgzh** [kiteretro/Circuit-Sword#85]
The temperatures for fan on and off are hard-coded to 60°C/55°. It would be nice to have this as a user-configurable setting somewhere.
Maybe also only allow reasonable settings here, e.g. on temperature max 70/75°C and off temperature must be below on temperature at least 5°C.

**Comment by timemaster5** [comment]
How is the fan wired, can I use a PWM on control pin? But anyway, configurable thresholds would be nice to have.

**Comment by timemaster5** [comment]
Ok, if I am right, the fan is connected to rpi pin 35 and there is no pwm.

**Comment by kiteretro** [comment]
Correct, you can probably use a 1k potentiometer in line with VCC of the fan and adjust to suit speed.
